posts/2023-01-11-hacktoberfest/index.qmd

```diff
@@ -15,15 +16,15 @@ categories:
 ## What is Hacktoberfest?
 
 In October last year, Bioconductor participated in [Hacktoberfest](https://hacktoberfest.com/).
-Hacktoberfest is a worldwide celebration of open-source projects that’s held during the month of October.
-To take part people had to [register](https://hacktoberfest.com/auth/) before Oct 31,
-have 4 pull/merge requests accepted between Oct 1 and Oct 31.
-The 1st 40,000 participants (maintainers & contributors) who complete Hacktoberfest could choose either a tree planted in their name or a t-shirt.
+Hacktoberfest is a worldwide celebration of open-source projects that is held during the month of October.
+To take part people had to [register](https://hacktoberfest.com/auth/) before October, 31<sup>st</sup>,
```

Comment:
```suggestion
To take part, people had to [register](https://hacktoberfest.com/auth/) before October, 31<sup>st</sup>,
```

Reply:
Fixed in [c776afc](https://github.com/Bioconductor/biocblog/pull/27/commits/c776afc595e5c6a730c2bceba54ba66813d4a089)